vehicle/mercedes/identity.go

```diff
@@ -65,21 +65,20 @@ func NewIdentity(log *util.Logger, token *oauth2.Token, account string, region s
 	}
 
 	if !token.Valid() {
-		v.log.DEBUG.Println("identity.NewIdentity - token not valid - Add expiry")
 		token.Expiry = time.Now().Add(time.Duration(10) * time.Second)
 	}
 
 	// database token
 	if !token.Valid() {
-		v.log.DEBUG.Println("identity.NewIdentity - token not valid - database token check started")
 		var tok oauth2.Token
 		if err := settings.Json(v.settingsKey(), &tok); err == nil {
+			v.log.DEBUG.Println("identity.NewIdentity - database token found")
 			token = &tok
 		}
 	}
 
 	if !token.Valid() && token.RefreshToken != "" {
-		v.log.DEBUG.Println("identity.NewIdentity - token not valid - refreshToken started")
+		v.log.DEBUG.Println("identity.NewIdentity - refreshToken started")
```

Comment:
Yes, this would indicate that a refreshtoken is present in db or yaml. If we only get the error in the next step then we can not divide between the cases that the refreshtoken is wrong or the case that the mbapi did not delivered a valid refreshtoken. (we had this beginning of last year)

Reply:
Genau das siehst Du ja im Tracelog am Aufruf danach.

Reply:
erklär das mal dem Nutzer :-) - Danke für das mergen...